web/shared/src/typeahead.ts

```diff
@@ -193,14 +194,25 @@ export const parse_unicode_emoji_code = (code: string): string =>
         .map((hex) => String.fromCodePoint(Number.parseInt(hex, 16)))
         .join("");
 
-export function get_emoji_matcher(query: string): (emoji: Emoji) => boolean {
+// TODO: can this file not import from web/src?
```

Comment:
I was hoping this could import from `web/src/composebox_typeahead` where this is defined

```js
type EmojiDict = {
    reaction_type: string;
    emoji_name: string;
    emoji_url?: string;
    emoji_code?: string;
    is_realm_emoji: boolean;
};
type EmojiPillData = EmojiDict & {type: "emoji"};
```

But VSCode didn't give me the automated fix, and I noticed there were no imports from `web/src` at the top of this file.